packages/client/hmi-client/src/components/widgets/tera-chart-settings.vue

```diff
@@ -11,6 +11,35 @@
 			:variables="selectOptions"
 			@configuration-change="$emit('selection-change', $event.selectedVariable, type)"
 		/>
+		<template v-if="type === ChartSettingType.SENSITIVITY && sensitivityOptions">
+			<!--FIXME: It might be better to move these inside the panel so that they can be controlled at an individual chart settings level -->
+
+			<label>Select parameter(s) of interest</label>
+			<MultiSelect
+				:disabled="_.isEmpty(selectedOptions)"
+				:placeholder="_.isEmpty(selectedOptions) ? 'Select variable(s) to display above' : 'Select parameters'"
```

Comment:
The placeholder message is confusing, I'd take out the "select variables to display above part". 

If you need instructional text for the users, put them outside.

Reply:
Ive removed this, this was here before I added the label